src/editors/playgroundController.ts

```diff
@@ -159,6 +158,8 @@ export default class PlaygroundController {
       }
     );
 
+    languageServerController._consoleOutputChannel = this._outputChannel;
```

Comment:
Now that the language server controller is outputting to the console, the `_outputChannel` in `playgroundController` is only used when clearing the output before evaluating. How do you feel about removing the `_outputChannel` here so it's only in the `languageServerController`? We can `outputChannel.clear()` in the `evaluate` function in the `languageServerController`.

Reply:
Good point. Makes sense to move the `_outputChannel` to the `LanguageServerController`. Do you think the current channel name `Playground output` is still valid?

Reply:
Have moved output channel to `LanguageServerController`.

Reply:
Looking at how folks see this:
![Screenshot 2024-01-03 at 1 15 08 PM](https://github.com/mongodb-js/vscode/assets/1791149/a7d8c67f-2f0c-4e33-9c90-c3e3ca766e90)

Maybe we should make it named something MongoDB specific, maybe `MongoDB Language Server Output`?
Or just `MongoDB Output` to keep it simple?
also fine as is, feel free to ignore.